src/stage-release.sh

```diff
@@ -38,12 +38,6 @@ cp $PULSAR_PATH/distribution/offloaders/target/apache-pulsar-offloaders-$VERSION
 
 cp -r $PULSAR_PATH/distribution/io/target/apache-pulsar-io-connectors-$VERSION-bin $DEST_PATH/connectors
 
-mkdir $DEST_PATH/RPMS
-cp -r $PULSAR_PATH/pulsar-client-cpp/pkg/rpm/RPMS/x86_64/* $DEST_PATH/RPMS
-
-mkdir $DEST_PATH/DEB
-cp -r $PULSAR_PATH/pulsar-client-cpp/pkg/deb/BUILD/DEB/* $DEST_PATH/DEB
```

Comment:
Perhaps file a subtask under #17724 about correspondingly update release guide.

Reply:
Yes, we'll remove the C++ release step from the main Pulsar release guide.